### ISSUE TITLE
Missing pips when using ranges with decimals.

### ISSUE THREAD

**Issue by CrouchingMuppet** [leongersen/noUiSlider#528]
It would appear that floating point variance is breaking the rendering of pips when using a range with decimal places, such as this fiddle: http://jsfiddle.net/a36r6m0d/

I use the following diff to patch, but it's possible there is a neater solution.
``` diff
--- assets/nouislider/distribute/nouislider.js
+++ assets/nouislider/distribute/nouislider.js
@@ -942,8 +942,12 @@ function closure ( target, options ){
 				return;
 			}
 
+			var fixFloatingPointVariance = function(value1, value2) {
+				return (value1 + value2).toFixed(7) / 1;
+			}
+
 			// Find all steps in the subrange.
-			for ( i = low; i <= high; i += step ) {
+			for ( i = low; i <= high; i = fixFloatingPointVariance(i, step) ) {
 
 				// Get the percentage value for the current step,
 				// calculate the size for the subrange.
```

**Comment by leongersen** [comment]
That indeed looks broken. If you'd like, you could make a pull request of your changes, or I could add them in myself. We could convert the example (thanks!) into a unit test as well.

**Comment by leongersen** [comment]
Fixed in 8.1.0; I've merged #528. Thanks for contributing!
